test/infamy/topology.py

```diff
@@ -39,7 +52,7 @@ def __init__(self, dotg):
             self.g.add_node(dn, kind=dk)
             self.g.add_edge(dn, d, weight=0)
 
-    def map_to(self, phy, node_match=match_kind):
+    def map_to(self, phy, node_match=match_node):
```

Comment:
We should _either_ remove the `node_match` parameter, and statically use `match_node` as an argument to the `GraphMatcher`, _or_ add an `edge_match` parameter here and set it to `match_edge` by default. Not this mix that we have now.

I would go for the first option. If we need it configurable in the future, we can add it back. Note: this was my mistake originally, sorry about that.

Reply:
fixed